src/resources/Correction/CCC.ts

```diff
@@ -18,10 +18,10 @@ export const CccCorrectionResource: ResourceIF = {
     priority: false
   },
   changeData: {
-    nameChangeOptions: [
-      NameChangeOptions.CORRECT_NEW_NR,
-      NameChangeOptions.CORRECT_NAME_TO_NUMBER,
-      NameChangeOptions.CORRECT_NAME
+    correctNameOptions: [
+      CorrectNameOptions.CORRECT_NEW_NR,
+      CorrectNameOptions.CORRECT_NAME_TO_NUMBER,
+      CorrectNameOptions.CORRECT_NAME
```

Comment:
I tested this:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/04e27344-f4e6-428e-9766-a6dd8fb1651c)